src/blockchain/sync-augur-node-with-blockchain.ts

```diff
@@ -14,14 +14,14 @@ export function syncAugurNodeWithBlockchain(db: Knex,  augur: Augur, ethereumNod
     startAugurListeners(db, augur, (blockNumber: string) => {
       db.raw(`SELECT highest_block_number FROM blockchain_sync_history ORDER BY highest_block_number DESC LIMIT 1`)
         .asCallback( (err: Error|null, rows?: HighestBlockNumberRow[]) => {
```

Comment:
There's a space between the parens `.asCallback( (err: Error|null,` which seems like bad style to me -- does tslint have a rule to prohibit that?